cmake/tribits/doc/build_ref/TribitsBuildReferenceBody.rst

```diff
@@ -3347,6 +3347,58 @@ This will ensure that every package that builds correctly will get installed.
 (The default 'install' target aborts on the first file install failure.)
 
 
+Installation Testing
+====================
+
+The CMake project <Project> has built-in support for testing an installation
+of itself using its own tests and examples.  The way it works is to configure,
+build, and install just the libraries and header files using::
```

Comment:
Does double colon at the end of a line mean something in Markdown?  I see it a couple times in this PR.

Reply:
@mhoemmen asked:

> Does double colon at the end of a line mean something in Markdown? I see it a couple times in this PR.

This is reStructuredText (RST), not MarkDown.  In RST, the double '::' means to show in as raw text.  This is equivalent to using:

```
  raw text
```

in Markdown.

Reply:
Thanks Ross!